### DISCUSSION TITLE
timestamps of messages, but missing the message itself

### DISCUSSION THREAD

**Discussion by Jeradin** [ReagentX/imessage-exporter#396] [Q&A] [answered]
Your script is really helpful! I was hoping to use it to find where the missing text is from past messages that show in my iMessage as "still typing" icon or just all the message timestamps displaying but no message text/body.

Have you seen this before?

![thread1](https://github.com/user-attachments/assets/4e30641c-fd89-4154-8005-3fdce3441646)
![thread2](https://github.com/user-attachments/assets/f68b2376-3040-490b-9ba3-c74cae06de58)

**Comment by ReagentX** [comment]
For messages from a database this old, the content may be stored in a different way than current databases or be missing entirely. Can you post a screenshot of how Messages.app displays them?

**Comment by Jeradin** [comment]
Thanks for the swift reply and moving this to a discussion!

Here is how it looks in my messages app.
![messages-thread](https://github.com/user-attachments/assets/816fea05-aa66-4782-9db0-b5269cf7ca22)

**Comment by ReagentX** [comment, marked as answer]
There is probably no data associated with the message body for those messages, but the rows still exist in the `message` table. Without seeing the rows it is difficult to say exactly what went wrong.

**Comment by Jeradin** [comment]
Thank you again @ReagentX for the support and replies. do you think it is worth me playing with the sqlite DB to try to find these missing messages or some other route?
